typeset/converter.js

```diff
@@ -108,11 +108,12 @@ const injectMathJax = async (log, inputPath, cssPath, outputPath, mathJaxPath) =
 
   // Typeset equations
   log.info(`Injecting MathJax (and typesetting)...`)
-  const didMathJaxLoad = await page.evaluate(/* istanbul ignore next */(mathJaxPath) => {
+  const didMathJaxLoad = await page.evaluate(/* istanbul ignore next */(mathJaxPath, mathJaxOutputFormat) => {
     console.log('Setting config for MathJax...')
+    console.log(`Setting output format to ${mathJaxOutputFormat}`)
     const MATHJAX_CONFIG = {
       extensions: ['mml2jax.js', 'MatchWebFonts.js'],
-      jax: ['input/MathML', 'output/HTML-CSS'],
+      jax: ['input/MathML', `output/${mathJaxOutputFormat}`],
```

Comment:
Does this mean the command-line argument is case-sensitive? And should we validate the string that is passed in?

When I ran it with `--format svg` I started seeing `browser-console Progress: "Loading [MathJax]/jax/output/HTML-CSS/fonts/STIX/...` messages which seemed to indicate that `HTML-CSS` was being used.

Reply:
It's an URL (in this case backed by a local filesystem), so yes, it is case sensitive.

I believe all MathJax output plugins have uppercase names, so we could just add `.toUpperCase()` there.